templates/cdhpages/blocks/accordion_block.html

```diff
@@ -1,5 +1,13 @@
 {% load wagtailcore_tags %}
 
+{% if self.heading %}
+{% include 'springkit/blocks/heading/jumplinkable_heading.html' %}
```

Comment:
Not vital, more curious,  does this work if you instead do `{% include_block self.heading %}`

Reply:
oh yeah it does :D